workflow/snakemake_rules/main_workflow.smk

```diff
@@ -1,58 +1,57 @@
-rule download:
-    message: "Downloading metadata and fasta files from S3"
+rule download_sequences:
+    message: "Downloading sequences from S3 bucket {params.s3_bucket}"
     output:
-        sequences = config["sequences"],
-        metadata = config["metadata"]
+        sequences = config["sequences"]
     conda: config["conda_environment"]
+    params:
+        s3_bucket = config["S3_BUCKET"]
     shell:
         """
-        aws s3 cp s3://nextstrain-ncov-private/metadata.tsv.gz - | gunzip -cq >{output.metadata:q}
-        aws s3 cp s3://nextstrain-ncov-private/sequences.fasta.gz - | gunzip -cq > {output.sequences:q}
+        aws s3 cp s3://{params.s3_bucket}/sequences.fasta.gz - | gunzip -cq > {output.sequences:q}
         """
 
-rule filter:
-    message:
-        """
-        Filtering to
-          - excluding strains in {input.exclude}
-        """
-    input:
-        sequences = rules.download.output.sequences,
-        metadata = rules.download.output.metadata,
-        include = config["files"]["include"],
-        exclude = config["files"]["exclude"]
+rule download_metadata:
+    message: "Downloading metadata from S3 bucket {params.s3_bucket}"
     output:
-        sequences = "results/filtered.fasta"
-    log:
-        "logs/filtered.txt"
-    params:
-        min_length = config["filter"]["min_length"],
-        exclude_where = config["filter"]["exclude_where"],
-        min_date = config["filter"]["min_date"],
-        date = date.today().strftime("%Y-%m-%d")
+        metadata = config["metadata"]
     conda: config["conda_environment"]
+    params:
+        s3_bucket = config["S3_BUCKET"]
     shell:
         """
-        augur filter \
-            --sequences {input.sequences} \
-            --metadata {input.metadata} \
-            --include {input.include} \
-            --max-date {params.date} \
-            --min-date {params.min_date} \
-            --exclude {input.exclude} \
-            --exclude-where {params.exclude_where}\
-            --min-length {params.min_length} \
-            --output {output.sequences} 2>&1 | tee {log}
+        aws s3 cp s3://{params.s3_bucket}/metadata.tsv.gz - | gunzip -cq >{output.metadata:q}
         """
 
+rule upload:
```

Comment:
Is our intention to have other users run these rules (by specifying their own buckets), or is this nextstrain-only functionality?

If it's nextstrain only, then I would advocate moving this rule to a snakemake file in `nextstrain_profiles/preprocess`. 

Similarly, if the downloading of preprocessed files is nextstrain only, I think it'd be simpler for them to live in `nextstrain_profiles/nextstrain`. This would make the pipeline conceptually easier to understand, and the default workflows wouldn't have to worry about ruleorders etc.

Reply:
that is a good idea. We can move the up and download rules to `local_rule` files.